tck/TCK_NOTES.adoc

```diff
@@ -0,0 +1,113 @@
+= Named Graphs =
+
+The TCK relies on a number of included named graphs.
+
+
+== Metadata File ==
+
+Each named graph is declared using a JSON file that references various Cypher script files for importing the graph as well as includes some additional statistical information.
+
+This JSON file is called the metadata file.
+
+=== Statistics Generation ===
+
+The following sections contain queries for the original Neo4j implementation of Cypher that have been used to generate the statistical information for some of the named graphs used in the TCK.
+
+These queries use the APOC procedure library for the generation of JSON results.
+
+=== Advices ===
+
+Some of the statistical information is annotated with additional advices (strings) that describes valid inferences that may be drawn from it in the context of how the given named graph is used in the TCK.
```

Comment:
"advices"? Ought this not to be "advice"? Is this some special term, given the context? If yes, I would put it into quotes.